setup.py

```diff
@@ -25,7 +25,7 @@
     include_package_data=True,
     package_data={
         "": [
-            "mlff/sph_ops/cgmatrix.npz",
+            "sph_ops/cgmatrix.npz",
             "tests/test_data/test_solid.npz",
```

Comment:
@thorben-frank are these test files needed? I'm not sure if they can be included this way because the package folder is the `mlff` folder and I assume the test files would need to be in there (but I'm not entirely sure)

Reply:
I ran a quick unit test in an extra branch [here](https://github.com/thorben-frank/mlff/tree/test_package_data) and seems that removing the test files does not break anything .. I feel like this was needed back in the days at some point but this seems not to be the case anymore. So I guess they can be deleted ...

Reply:
ok, removed